sdk/sdk-api/src/main/java/mil/emp3/api/utils/GeoLibrary.java

```diff
@@ -44,16 +43,16 @@ public static double getEarthRadiusAt(double Latitude) {
      * @param oLatLon1
      * @param oLatLon2
      * @return The distance in meters.
-     * @throws InvalidParameterException
+     * @throws IllegalArgumentException
      */
-    public static double computeDistanceBetween(IGeoPosition oLatLon1, IGeoPosition oLatLon2) throws InvalidParameterException {
+    public static double computeDistanceBetween(IGeoPosition oLatLon1, IGeoPosition oLatLon2) throws IllegalArgumentException {
```

Comment:
Remove IllegalArgumentException from throws clause but document why it may be thrown.